coder4097/todo-backend
# Update added todo item feature

## Description

Update a TODO item:

URI: PUT /todos/:id
req.body: { "title": "title" }
res.status: 200
res: { "id": "id", "title": "title" }

### Database schema changes
None 

### Tests

### Automated test cases added

- Returns 200 if a todo item exists and the title passed in body is valid.
- Return 404 if todo item doesn't exists.
- Return 400 if the title passed in the req body is either empty or not a string.

### Manual test cases run
req.body = {
    "title":"updated title for check"
}

res = {
    "id": "6114b0a5831b7c11f0306e64",
    "title": "updated title for check"
}
